bindings/python/convert.py

```diff
@@ -101,12 +157,12 @@ def convert_multi(model_id: str, folder: str, token: Optional[str]) -> Conversio
     return operations, errors
 
 
-def convert_single(model_id: str, folder: str, token: Optional[str]) -> ConversionResult:
+def convert_single(model_id: str, *, revision: Optional[str], folder: str, token: Optional[str], discard_names: List[str]) -> ConversionResult:
     pt_filename = hf_hub_download(repo_id=model_id, filename="pytorch_model.bin", token=token, cache_dir=folder)
```

Comment:
Use `revision` parameter to download `pt_filename`?

Reply:
Forgot ! Thanks